audit/pkg/command/health.go

```diff
@@ -0,0 +1,18 @@
+package command
+
+import (
+	"github.com/owncloud/ocis/audit/pkg/config"
+	"github.com/urfave/cli/v2"
+)
+
+// Health is the entrypoint for the health command.
+func Health(cfg *config.Config) *cli.Command {
+	return &cli.Command{
+		Name:  "health",
+		Usage: "Check health status",
+		Action: func(c *cli.Context) error {
+			// Not implemented
+			return nil
```

Comment:
Can we 1) either give a basic health status back or  2) create a ticket to implement it in a separate task